src/mondeath.cpp

```diff
@@ -319,7 +319,7 @@ void mdeath::fungus( monster &z )
     sounds::sound( z.pos(), 10, _( "Pouf!" ) );
 
     fungal_effects fe( *g, g->m );
-    for( auto &&sporep : g->m.points_in_radius( z.pos(), 1 ) ) {
+    for( auto &&sporep : g->m.points_in_radius( z.pos(), 1 ) ) { // *NOPAD*
```

Comment:
Would replacing it with `for( const tripoint &sporep :` work better? It may not confsue `astyle` and it's clearer to read anyway.

Reply:
I want to replace all instances of `auto` with proper type names later in separate PR(s), so I will check if it helps.

Reply:
If you want a minimal change that avoids the astyle formatting issue you can change to `const auto &`